box2d3-wasm/README.md

```diff
@@ -1,30 +1,30 @@
 # box2d3-wasm
 
-Let's try and use [Box2D v3][] on the web via [wasm][]!
+[Box2D v3][] ported to the web via [wasm][]!
 
-```bash
-git clone --recurse-submodules https://github.com/Birch-san/box2d3-wasm.git
-```
+## Online Demos
 
-## Install from npm
+Check out the [pyramid drop demo](https://box2d-v3-threading.netlify.app/?threading=1&pyramidHeight=64). You can change the size of the pyramid by editing the "pyramidHeight" URL parameter. Set it to 101 for a pyramid with 5,050 boxes!
 
-Install [Node.js and npm][].  
-Install [npm package](https://www.npmjs.com/package/box2d3-wasm) like so:
+Check out the [stress test demo](https://phaser-stress-test-box2d3-wasm.netlify.app/?threading=1). How many shapes can your computer simulate at 60fps? box2d3-wasm is the fastest Box2D implementation on the web.
```

Comment:
I wanna be more dignified than this. perf should speak for itself, we don't need to hype it up with invitations or claims. we should prefer to state facts (like we have support for SIMD* and threading) without diminishing other people's efforts. if we have benchmarks we can point at those numbers, but I don't want a tone of getting overexcited at our own results.

*note: we haven't really verified whether the SIMD support is working as intended, so best not to oversell this until we are more confident it's integrated correctly. the previous box2d-wasm got I think <1% speedup from SIMD, and there was evidence that only a small amount of the code was getting vectorized. we're likely to be in a much better situation than this now that box2d is designed for SIMD, but worth measuring first.

Reply:
alright for sure